contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -60,7 +69,18 @@ def main(argv):
         VerifyTracksExist(track_names=["Page*", "*System*", "*CGroup*"], allow_duplicates=True),
 
         AddIterator(function_name="TestFunc2"),
-        VerifyFunctionCallCount(function_name='TestFunc2', min_calls=1257, max_calls=1257)
+        VerifyFunctionCallCount(function_name="TestFunc2", min_calls=1257, max_calls=1257),
+
+        # Let's take a capture of the current version and verify this can be still loaded
+        EndSession(),
+        ConnectToStadiaInstance(),
+        FilterAndSelectFirstProcess(process_filter="hello_ggp"),
+        Capture(),
+        VerifyTracksExist(track_names="hello_ggp_stand*", allow_duplicates=True),
+        EndSession(),
+        # If we took the capture around midnight, we need to ensure to also look for the next day
```

Comment:
Nope, I thought the same as well at the beginning, but note, that those two string constants are generated BEFORE the test is run. So the `date.today()` will be definitly before the capture gets taken/saved.

Reply:
I also stumbled about this.
The timestamps are taken at the start of the test; so the relevant days are today and tomorrow (from the perspective of the test start time)

Reply:
Oh, ok, that makes sense, but is really hard to understand without an explanation. Please add a comment about this. Alternative idea to test this: You could also change the `LoadLatestCapture` test to not receive a filter string, but an executable name and a timerange. It could then sort the list of captures by date, filter by executable name, and verify if the top capture is no older then the specified timerange - you should be able to parse the date from the name and convert it to `datetime`.

Reply:
Added a lengthy comment.